src/parser.js

```diff
@@ -21,9 +21,7 @@ function parse(text, parsers, options) {
       if (ctx.initExpression) {
         ctx.initExpression.omitSemicolon = true;
       }
-      if (ctx.loopExpression) {
-        ctx.loopExpression.omitSemicolon = true;
-      }
+      ctx.loopExpression.omitSemicolon = true;
```

Comment:
I guess that w/in `ForStatement`s we always have a `ctx.loopExpression` right?

Reply:
Yeah, the initial expression and the condition expression can be `null` but for some reason the loop expression is an expression with `null` content.
Maybe we can ask for some consistency but right now that’s how it is